### ISSUE TITLE
M4 MLP is Linear Regression

### ISSUE THREAD

**Issue by sjtrny** [forecastlab/forecast_dash#96]
From updater/models.py

```
        self.model = MLPRegressor(
            hidden_layer_sizes=6,
            activation="identity",
            solver="adam",
            max_iter=100,
            learning_rate="adaptive",
            learning_rate_init=0.001,
            random_state=42,
        )
```

The `activation="identity"` parameter means that no activation function is applied and this network can be replaced by a single identity layer.

This raises some questions:
- should we use a non-linear activation function (sigmoid, tanh or relu)?
- should we just replace this with linear regression?
- should we remove it entirely?

It takes a surprisingly long time to train these models so I think it is worthwhile to investigate alternatives.

**Comment by sjtrny** [comment]
This is even more important given that these models cause training to take hours. It's 10:30am and the models are still running from the night before (0300 start).

**Comment by sjtrny** [comment]
Resolved by https://github.com/forecastlab/forecast_dash/pull/100
